userguide/snippets/x_drivetrays.rst

```diff
@@ -25,7 +25,7 @@ next drive tray to the right of the first. Install remaining drive
 trays to the right across the row. After a row is filled with drives,
 move down to the next row and start again with the left bay.
 
-This example shows the proper order for a SLOG SSD, an L2ARC SSD,
+This example shows the proper order for a W SSD, an R SSD,
```

Comment:
This should be similar to above.  The W and R labels are ambiguous on their own.
`` This example shows the proper order for a write cache (:literal:`W`) SSD, a read cache (:literal:`R`) SDD, ``